src/main/java/no/fint/antlr/odata/ODataOperator.java

```diff
@@ -8,14 +8,7 @@ public class ODataOperator {
 
     public static Boolean eq(Object property, String value) {
```

Comment:
hmm... Har du blanda litt her. eq metoden returnerer startswith og ikke equalsIgnoreCase. Eller er dette bare for test

Reply:
Det var for å teste at koden faktisk fungerte, siden startwith metoden blir aldri kalt på. mente ikke å comitten den endringen 😅